EIPS/eip-2315.md

```diff
@@ -62,14 +62,14 @@ _Note 3: The description above lays out the semantics of this feature in terms o
 
 ### Indirect Jumps
 
-If [EIP-3337 BEGINDATA](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-3337.md) is implemented then the indirect jumps from EIP-615 -- `JUMPV` and `JUMPSUBV` -- can be implemented.  These would take two arguments on the stack: a constant offset relative to `BEGINDATA` to a jump table, and a variable index into that table.  Detailed specifications can await the acceptance EIP-3337.
+If [EIP-3337 BEGINDATA](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-3337.md) is implemented then the indirect jumps from  [EIP-615](https://eips.ethereum.org/EIPS/eip-615) -- `JUMPV` and `JUMPSUBV` -- can be implemented.  These would take two arguments on the stack: a constant offset relative to `BEGINDATA` to a jump table, and a variable index into that table.  Detailed specifications can await the acceptance EIP-3337.
```

Comment:
This should not be merged as Review with a link to 615 until 615 is in review.  Either remove references to 615 from this document (so it is no longer dependent on it) or push to get 615 to review, or move this back to Draft.